### PR TITLE
Some WOPI fixes

### PR DESCRIPTION
**Pull request by mmattel**
Some fixes to the WOPI documentation.

Backport to 10.6 and 10.7

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 1]
```diff
@@ -52,52 +54,42 @@ Microsoft Office Online access can be restricted to users in a specific group, b
...
 In the example above, only users in the `admin` group would be able to access Microsoft Office Online.
 
-NOTE: If the key is not defined, then all users have access to WOPI.
+NOTE: If the key is not defined, then all users have access to this Microsoft Office Online service connected via WOPI.
 
 == Locking the Document
 
-If you open a document with Microsoft Office Online in ownCloud, it makes use of the WebDAV file locking functionality, available in ownCloud server. 
-The idea is to lock the file so other users, who have access, can't make changes to the document while you're editing it. 
+If you open a document with Microsoft Office Online in ownCloud, it makes use of the WebDAV file locking functionality, available in ownCloud server. The idea is to lock the file so other users, who have access, can't make changes to the document while you're editing it.
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:61]:
```suggestion
If you open a document with Microsoft Office Online in ownCloud, it makes use of the WebDAV file locking functionality available in ownCloud server. The idea is to lock the file so other users with access can't make changes to the document while you're editing it. 
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 2]
```diff
@@ -52,52 +54,42 @@ Microsoft Office Online access can be restricted to users in a specific group, b
...
-The idea is to lock the file so other users, who have access, can't make changes to the document while you're editing it. 
+If you open a document with Microsoft Office Online in ownCloud, it makes use of the WebDAV file locking functionality, available in ownCloud server. The idea is to lock the file so other users, who have access, can't make changes to the document while you're editing it. 
 
-In other words: the feature ensures that you are the "master editor", your changes will always be the "master state". 
-Other users can make changes, e.g., with the desktop client, but those will create conflict files for them, which can be resolved afterward.
-When you close the document, Microsoft Office Online unlocks the file, so others can edit it.
+In other words: the feature ensures that you are the "master editor", your changes will always be the "master state".  Other users can make changes, e.g., with the desktop client, but those will create conflict files for them, which can be resolved afterward. When you close the document, Microsoft Office Online unlocks the file, so others can edit it.
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:63]:
```suggestion
In other words, the feature ensures that you are the "master editor". Your changes will always be the "master state".  Other users can make changes, e.g., with the desktop client, but those will create conflict files for them, which can be resolved afterward. When you close the document, Microsoft Office Online unlocks the file so others can edit it.
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 3]
```diff
@@ -52,52 +54,42 @@ Microsoft Office Online access can be restricted to users in a specific group, b
...
 
 You can always click on the lock icon next to your file name and unlock it manually using the button in the sidebar.
 
 === Lock Timeout
 
-If a user is editing a file and loses their internet connection the lock will timeout, freeing the lock, after 30 minutes. 
-Please refer to {wopi-timeout-documentation-url}[the WOPI documentation] for further information.
+If a user is editing a file and loses their internet connection the lock will timeout, freeing the lock, after 30 minutes. Please refer to {wopi-timeout-documentation-url}[the WOPI documentation] for further information.
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:69]:
```suggestion
If a user is editing a file and loses their internet connection, the lock will timeout, freeing the lock after 30 minutes. Refer to {wopi-timeout-documentation-url}[the WOPI documentation] for further information.
```

**Review by EParzefall — APPROVED** [review]
Just some punctuation and style issues.

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 4]
```diff
@@ -52,52 +54,42 @@ Microsoft Office Online access can be restricted to users in a specific group, b
...
 Checklist if something is not working:
 
-1. **Client** can reach the **ownCloud Server** (browse to web page und log in)
-
-2. **Client** can reach the **Office Online Server** (via hosting/discovery url with https)
-
-3. **ownCloud Server** can reach the **Office Online Server** (via hosting/discovery url with https)
-
-4. **Office Online Server** can reach **ownCloud Server** (browse to web page und log in)
+. **Client** can reach the **ownCloud Server** (browse to web page und log in)
```
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:86]:
```suggestion
. **Client** can reach the **ownCloud Server** (browse to web page and log in)
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 5]
```diff
@@ -52,52 +54,42 @@ Microsoft Office Online access can be restricted to users in a specific group, b
...
+. **Client** can reach the **Office Online Server** (via hosting/discovery url with https)
+. **ownCloud Server** can reach the **Office Online Server** (via hosting/discovery url with https)
+. **Office Online Server** can reach **ownCloud Server** (browse to web page und log in)
```
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:89]:
```suggestion
. **Office Online Server** can reach **ownCloud Server** (browse to web page and log in)
```

**Review by phil-davis — COMMENTED** [review]
While this is being edited, we may as well change "und" to "and"

**Comment by EParzefall** [comment]
> While this is being edited, we may as well change "und" to "and"

Definitely! Far too easy for German speakers to overlook.

**Pull request merged**